runelite-client/src/main/java/net/runelite/client/plugins/xpupdater/XpUpdaterPlugin.java

```diff
@@ -166,6 +168,29 @@ private void update(String username)
 
 			sendRequest("TempleOSRS", request);
 		}
+
+		if (config.wiseoldman())
+		{
+			HttpUrl url = new HttpUrl.Builder()
+					.scheme("https")
+					.host("wiseoldman.net")
+					.addPathSegment("api")
+					.addPathSegment("players")
+					.addPathSegment("track")
+					.build();
```

Comment:
All of your chained calls here are indented one level too far.
```suggestion
				.scheme("https")
				.host("wiseoldman.net")
				.addPathSegment("api")
				.addPathSegment("players")
				.addPathSegment("track")
				.build();
```